recipe/meta.yaml

```diff
@@ -12,19 +12,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skip: True # [py<37]
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - pip
-    - setuptools >=42
-    - setuptools_scm >=3.4
+    - setuptools_scm
     - wheel
```

Comment:
`toml` is missing. It's inside brackets https://github.com/colour-science/flask-compress/blob/079b2152c9a40c5eaad4a28491f384ccae6ab6c5/pyproject.toml#L2. `setuptools_scm[toml]` means `setuptools_scm` **with** `toml`

Reply:
`setuptools_scm[toml]` should be:
```
- setuptools_scm
- toml
```